src/keylessBackup/index.ts

```diff
@@ -28,6 +30,7 @@ export async function storeEncryptedMnemonic({
     body: JSON.stringify({
       encryptedMnemonic,
       encryptionAddress,
+      token: jwt,
```

Comment:
I think JWTs are normally given in an auth header

Reply:
Usually yes, @satish-ravi commented something similar [here](https://github.com/valora-inc/cloud-account-backup/pull/158#discussion_r1462537029), but since we're not using as an auth token and instead as a one time token and I opted to keep it in the body. This avoids having to setup middleware for the backend for one endpoint; Also IIRC there was also some default handling of jwt's in as auth header where it was passed to subsequent requests that was also undesirable.